pkgs/development/python-modules/py-multihash/default.nix

```
@@ -0,0 +1,46 @@
+{ base58
+, buildPythonPackage
+, fetchPypi
+, lib
+, morphys
+, pytest
+, pytestcov
+, pytestrunner
+, six
+, variants
+, varint
+}:
+buildPythonPackage rec {
+  pname = "py-multihash";
+  version = "0.2.3";
```

I didn't investigate this very thoroughly, but it looks like python2 is unable to import any of the packages
```suggestion
  version = "0.2.3";
  disabled = isPy27;
```

```
builder for '/nix/store/hxf2k0jvfnz3gqqz1gsrnjhgn0l7z6rj-python2.7-py-multihash-0.2.3.drv' failed with exit code 2; last 10 log lines:
  Traceback:
  tests/test_multihash.py:11: in <module>
      from multihash import (
  multihash/__init__.py:9: in <module>
      from .multihash import (
  multihash/multihash.py:9: in <module>
      import multihash.constants as constants
  E   ImportError: No module named constants
  !!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!
  =========================== 1 error in 0.17 seconds ============================
cannot build derivation '/nix/store/92nbj7r7mkz9872mnbxpi9ziy23vr7ys-env.drv': 1 dependencies couldn't be built
[0 built (1 failed)]
error: build of '/nix/store/92nbj7r7mkz9872mnbxpi9ziy23vr7ys-env.drv' failed
https://github.com/NixOS/nixpkgs/pull/83203
1 package failed to build:
python27Packages.py-multihash

2 package built:
python37Packages.py-multihash python38Packages.py-multihash
```

---

done.